docs/rules/prefer-error-cause.md

```diff
@@ -0,0 +1,48 @@
+# Prefer specifying an old error as `cause` option where rethrowing the error
```

Comment:
```suggestion
# Prefer specifying an old error as `cause` option when rethrowing the error
```

Reply:
I think this could be better written. "old error" is not really the terminology I would use.

Reply:
> I think this could be better written. "old error" is not really the terminology I would use.

How about `caught error`?

```
Prefer specifying a caught error as `cause` option when rethrowing the error
```